### DISCUSSION TITLE
I don't have access to the free student CoPilot subscription

### DISCUSSION THREAD

**Discussion by medovanx** [community/community#31276] [Copilot Conversations] [answered]
Even tho it says Pro on my profile when I open VS Code, co-pilot says I need a subscription. it used to work before 
![Screenshot 2022-08-30 010008](https://user-images.githubusercontent.com/29468096/187313389-af0678dd-c85e-4629-aaac-cdbafad422b6.png)
![Screenshot 2022-08-30 010022](https://user-images.githubusercontent.com/29468096/187313408-3290dbbe-d636-41d4-97ef-3b1f81846daf.png)

**Comment by davecheney** [comment, marked as answer]
I’m sorry about the wording, this is a generic message from the VSCode marketplace. For you, if you have an active student coupon, you are eligible for a complementary copilot subscription.
